docs/reference/field-types/relationship-reverse.md

```diff
@@ -0,0 +1,35 @@
+# `relationshipReverse`
+
+A `relationshipReverse` field allows us to access the "other side" of a doc [relationship](/reference/field-types/relationship.md). There is no editing interface for this field type as the relationship is defined in the paired `relationship` field. `relationshipReverse` fields are a convenience allowing developers to identify Apostrophe documents that have established relationships with a given document.
+
+## Module field definition
+```javascript
+// Configuring the `_pizzas` field in a module's `fields.add` subsection:
+_pizzas: {
+  type: 'relationshipReverse',
+  withType: 'pizza',
+  reverseOf: '_fabrics'
+}
+```
+
+We can now see `_pizzas` as a property of each `topping` object that is related to a product.
+
+If desired, we can specify `schema` and `relationshipsField` just as we would for `joinByArray`. Currently these are not automatic in a reverse join and must be fully specified if relationship properties are to be accessed. Most array joins do not have relationship properties and thus do not require reverse access to them.
```

Comment:
`joinByArray` is no longer a thing. Also if this isn't automatic that's a bummer that should 100% get fixed, it makes me wonder if we should wait on documenting relationship-reverse until it is fixed. Because there's no possible valid value for these properties, other than what is already set on the relationship pointing back at us.

Reply:
> makes me wonder if we should wait on documenting relationship-reverse until it is fixed.

I wasn't clear about this, but that's what I did, which is why `joinByArray` is still there. I brought the page over and started in on it, but I couldn't test it successfully so I didn't update this page. You can see it commented out in the sidebar nav file so it's not findable. There's a ticket, A30U-983, to dig into the field type.

Reply:
Note added. I commented out the only link to this page.